### ISSUE TITLE
Tests fail on 0.21.6.30

### ISSUE THREAD

**Issue by TeddyDD** [andreafrancia/trash-cli#210]
**Describe the bug**
Tests fails on  version: 0.21.6.30

**trash-cli version**
0.21.6.30

**Operating system:**
 - OS: Void 5.12.14_1 x86_64 AuthenticAMD uptodate rrrrmmnFFFFFFFFF

**To Reproduce**
Copy and paste the commands (and their output) to execute in order to reproduce 
the behavior:

$ pip install -r requirements-dev.txt -r requirements.txt
$ pytest

<details>

```=============================================================================================== test session starts ================================================================================================
platform linux -- Python 3.9.6, pytest-6.2.4, py-1.10.0, pluggy-0.13.1
rootdir: /tmp/quic-tmp.o56xH/trash-cli
collected 274 items

tests/test_bump.py ...                                                                                                                                                                                       [  1%]
tests/test_fake_fstab.py ...                                                                                                                                                                                 [  2%]
tests/test_fake_ismount.py .......                                                                                                                                                                           [  4%]
tests/test_file_descriptions.py .......                                                                                                                                                                      [  7%]
tests/test_files.py ...                                                                                                                                                                                      [  8%]
tests/test_filesystem.py ......                                                                                                                                                                              [ 10%]
tests/test_issues_message.py .                                                                                                                                                                               [ 10%]
tests/test_joining_paths.py .                                                                                                                                                                                [ 11%]
tests/test_list_all_trashinfo_contents.py ..                                                                                                                                                                 [ 12%]
tests/test_make_script.py .....                                                                                                                                                                              [ 13%]
tests/test_parsing_trashinfo_contents.py ............                                                                                                                                                        [ 18%]
tests/test_restore_cmd.py ................                                                                                                                                                                   [ 24%]
tests/test_restore_trash.py .......                                                                                                                                                                          [ 26%]
tests/test_shrink_user.py .....                                                                                                                                                                              [ 28%]
tests/test_storing_paths.py ..                                                                                                                                                                               [ 29%]
tests/test_top_trash_dir_rules.py ....                                                                                                                                                                       [ 30%]
tests/test_trash_empty.py ....                                                                                                                                                                               [ 32%]
tests/test_trash_list.py ................                                                                                                                                                                    [ 37%]
tests/test_trash_put_reporter.py .                                                                                                                                                                           [ 38%]
tests/test_trash_put_slow.py .............                                                                                                                                                                   [ 43%]
tests/test_trash_rm.py .......                                                                                                                                                                               [ 45%]
tests/test_trash_rm_script.py ...                                                                                                                                                                            [ 46%]
tests/test_trash_rm_slow.py ....                                                                                                                                                                             [ 48%]
tests/test_trashdir.py ........                                                                                                                                                                              [ 51%]
tests/test_trashing_a_file.py ..                                                                                                                                                                             [ 51%]
tests/test_volume_of.py ...                                                                                                                                                                                  [ 52%]
tests/empty/test_clock.py ...                                                                                                                                                                                [ 54%]
tests/empty/test_empty_end_to_end.py ...........                                                                                                                                                             [ 58%]
tests/empty/test_empty_end_to_end_with_argument.py ....                                                                                                                                                      [ 59%]
tests/empty/test_parser.py ..........                                                                                                                                                                        [ 63%]
tests/list/test_deletion_date_extractor.py ..                                                                                                                                                                [ 63%]
tests/list/test_end_to_end_list.py ...                                                                                                                                                                       [ 64%]
tests/list/test_file_size.py .                                                                                                                                                                               [ 65%]
tests/list/test_trash_dirs_selector.py ..F                                                                                                                                                                   [ 66%]
tests/list/test_trash_list_parser.py ........                                                                                                                                                                [ 69%]
tests/put/test_atomic_write.py ....                                                                                                                                                                          [ 70%]
tests/put/test_create_trashinfo_basename.py ...                                                                                                                                                              [ 71%]
tests/put/test_end_to_end_put.py ........                                                                                                                                                                    [ 74%]
tests/put/test_file_trasher.py ..                                                                                                                                                                            [ 75%]
tests/put/test_get_option_parser.py ...........                                                                                                                                                              [ 79%]
tests/put/test_home_fallback.py ..                                                                                                                                                                           [ 80%]
tests/put/test_parent_path.py ....                                                                                                                                                                           [ 81%]
tests/put/test_persist_trash_info.py ..                                                                                                                                                                      [ 82%]
tests/put/test_suffix.py ...                                                                                                                                                                                 [ 83%]
tests/put/test_trash_directories_finder.py ..                                                                                                                                                                [ 84%]
tests/put/test_trasher.py ....                                                                                                                                                                               [ 85%]
tests/put/test_user.py ......                                                                                                                                                                                [ 87%]
tests/restore/test_all_trash_directories.py .                                                                                                                                                                [ 88%]
tests/restore/test_end_to_end_restore.py ....                                                                                                                                                                [ 89%]
tests/restore/test_is_trashed_from_path.py ....                                                                                                                                                              [ 91%]
tests/restore/test_parse_indexes.py ..........                                                                                                                                                               [ 94%]
tests/restore/test_restore_asking_the_user.py ..                                                                                                                                                             [ 95%]
tests/restore/test_trash_directories2.py ..                                                                                                                                                                  [ 96%]
tests/restore/test_trash_directory.py ...                                                                                                                                                                    [ 97%]
tests/trash_dir_scanner/test_top_trash_dir_rules.py ....                                                                                                                                                     [ 98%]
tests/trash_dir_scanner/test_trash_dir_scanner.py .                                                                                                                                                          [ 99%]
tests/trash_dir_scanner/test_user_info_provider.py ..                                                                                                                                                        [100%]

===================================================================================================== FAILURES =====================================================================================================
____________________________________________________________________________________ TestTrashDirsSelector.test_user_specified _____________________________________________________________________________________

self = <tests.list.test_trash_dirs_selector.TestTrashDirsSelector testMethod=test_user_specified>

    def test_user_specified(self):

        result = list(self.selector.select(False, ['user-specified-dirs']))

>       assert result == [(trash_dir_found, ('user-specified-dirs', '/'))]
E       AssertionError: assert [(trash_dir_f...rs', '/tmp'))] == [(trash_dir_f...-dirs', '/'))]
E         At index 0 diff: (trash_dir_found, ('user-specified-dirs', '/tmp')) != (trash_dir_found, ('user-specified-dirs', '/'))
E         Use -v to get the full diff

tests/list/test_trash_dirs_selector.py:22: AssertionError
============================================================================================= short test summary info ==============================================================================================
FAILED tests/list/test_trash_dirs_selector.py::TestTrashDirsSelector::test_user_specified - AssertionError: assert [(trash_dir_f...rs', '/tmp'))] == [(trash_dir_f...-dirs', '/'))]
========================================================================================== 1 failed, 273 passed in 1.76s ===========================================================================================
```
</details>

**Comment by andreafrancia** [comment]
Fixed in 0.21.7.23 (https://github.com/andreafrancia/trash-cli/releases/tag/0.21.7.23)

**Comment by TeddyDD** [comment]
I'm getting following error on 0.21.7.23 

```
tests/trash_dir_scanner/test_user_info_provider.py ..                    [100%]

=================================== FAILURES ===================================
_ Test_when_deleting_an_existing_file_in_verbose_mode.test_should_tell_where_a_file_is_trashed _

self = <tests.test_trash_put_slow.Test_when_deleting_an_existing_file_in_verbose_mode testMethod=test_should_tell_where_a_file_is_trashed>

    def test_should_tell_where_a_file_is_trashed(self):
        output = self.fixture.stderr.splitlines()
>       assert (("trash-put: '%s' trashed in %s/XDG_DATA_HOME/Trash" %
                 (self.foo_file, self.fixture.temp_dir)) in
                  output)
E       AssertionError: assert ("trash-put: '%s' trashed in %s/XDG_DATA_HOME/Trash" % ('/tmp/tmpwhioyenx/foo', '/tmp/tmpwhioyenx')) in ['trash-put: Volume of file: /', 'trash-put: Trash-dir: /tmp/tmpwhioyenx/XDG_DATA_HOME/Trash from volume: /', "trash-put: '/tmp/tmpwhioyenx/foo' trashed in ~/tmpwhioyenx/XDG_DATA_HOME/Trash"]

tests/test_trash_put_slow.py:64: AssertionError
=========================== short test summary info ============================
FAILED tests/test_trash_put_slow.py::Test_when_deleting_an_existing_file_in_verbose_mode::test_should_tell_where_a_file_is_trashed
======================== 1 failed, 265 passed in 1.74s =========================
```

Should I open separate issue?

**Comment by andreafrancia** [comment]
No, you don't need to open a separate issue.
Please can you give me the output of this command: `echo $HOME`?

**Comment by andreafrancia** [comment]
The second should be fixed in 0.21.7.24 (https://github.com/andreafrancia/trash-cli/releases/tag/0.21.7.24)
